docs/spec/executable/tes-http-serverd.md

```diff
@@ -6,39 +6,28 @@ credentials for other processes to acquire.
 For information on TES, see
 <https://docs.aws.amazon.com/greengrass/v2/developerguide/interact-with-aws-services.html>.
 
-The following is the current daemon implementation flow:
+The following are the requirements for the `tes-http-serverd`:
 
 - [tes-http-serverd-1] The `tes-http-serverd` daemon will use the `tes` daemon
-  for acquiring credentials.
-- [tes-http-serverd-2] On connection, the `tes-http-serverd` daemon will
-  validate the provided token and establish an http connection with the server.
+  for acquiring TES credentials.
+- [tes-http-serverd-2] The `tes-http-serverd` daemon will validate client connections
+  with the provided `AWS_CONTAINER_AUTHORIZATION_TOKEN` (see [tes-http-serverd-ev-#]).
 - [tes-http-serverd-3] On request, the `tes-http-serverd` daemon will validate
   the request and headers.
 - [tes-http-serverd-4] On request, the `tes-http-serverd` daemon will make a
   request to the `tes` daemon to acquire the temporary credentials vended, and
   return to the caller.
+- [tes-http-serverd-5] The CLI arguments shall be the optional argument `port`.
+- [tes-http-serverd-6] The `port` cli argument configures the tes-http-serverd TCP port.
+- [tes-http-serverd-7] If no `port` argument is supplied on the CLI, the value for `port` shall be `8090`.
```

Comment:
Should use random port, as provided by OS when 0 is passed for requested port.

Reply:
recipe runner will pass the ip/port to components

Reply:
Is that an environment variable? How does receipe runner pass the IP/port to the components?

Reply:
Env var, yes